docker-compose.yml

```diff
@@ -12,8 +12,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Installation Steps:
+# ------------------
+#
+# 1. Create the 'sodafoundation/delfin' docker image with Dockerfile in
+#    delfin project using command below. (Note: In future we will upload
+#    this image to docker-hub and it can be downloaded with out this step)
+#
+#    $ docker build -t sodafoundation/delfin .
```

Comment:
One generic question. when we do a docker pull from dcker hub, how do we provide the conf ?